kuma/wiki/templates/wiki/document.html

```diff
@@ -390,7 +397,7 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                 {% endif %}
 
                 <!-- approvals -->
-                {% if approvals_html %}
+		{% if approvals_html %}
```

Comment:
This one actually needs an extra space added to fix the indentation to match what is was before.